scripts/build.py

```diff
@@ -712,6 +725,14 @@ def minify_func(source_path, target_path, file_hashes, filename):
     elif filename.endswith('.css') or filename.endswith('.js'):
         print 'Minifying %s' % source_path
         _minify(source_path, target_path)
+    elif filename.endswith('.ts'):
+        compile_typescript_files(source_path)
+        target_path = target_path.replace('.ts', '.js')
+        minified_target_path = target_path.replace('.js', '_min.js')
```

Comment:
I think normally we go with .min.js (did you find evidence to the contrary)?

Also I notice we deal with '.js' files in the previous elif clause -- is that still needed?

Reply:
There are some files which are present as js files in the source dir, so yes we need that clause.